frontend/src/ui/page/muscles.rs

```diff
@@ -96,12 +93,12 @@ pub fn view(model: &Model, data_model: &data::Model) -> Node<Msg> {
                         domain::Muscle::description(*m)
                     ],
                     common::view_chart(
-                        &[("Set volume (weekly total)", common::COLOR_SET_VOLUME)],
+                        &[("Set volume (7 day total)", common::COLOR_SET_VOLUME)],
                         common::plot_chart(
                             &[common::PlotData {
-                                values: set_volume,
-                                plots: common::plot_line_with_dots(common::COLOR_SET_VOLUME),
-                                params: common::PlotParams::primary_range(0., 10.),
+                                values: total_7day_set_volume,
+                                plots: common::plot_line(common::COLOR_SET_VOLUME),
+                                params: common::PlotParams::default(),
```

Comment:
I think this chart should always start at 0 and we should use the same parameters as for the set volume chart on the Training page:

```suggestion
                                params: common::PlotParams::primary_range(0., 10.),
```